### ISSUE TITLE
App can't connect to Parse-Server after some hours.

### ISSUE THREAD

**Issue by mithleshjs** [ParsePlatform/parse-server#1070]
Parse-Server works fine for sometime but after that my android app can't connect to it but my PHP app can still communicate with it. Only after I restart the parse-server, app connects to the Parse-Server.

**Comment by gfosco** [comment]
There's not enough information here to diagnose a problem, especially if only your android app is unable to connect... I'm sorry but you'll need to do more troubleshooting and examine logs, and see if you can find a real issue here related to the server.  If and when you do find something, please open a new issue with relevant details.  Thanks.
